test/Test/JwtSpec.hs

```diff
@@ -1,35 +1,32 @@
-module Test.JWTSpec where
+module Test.JwtSpec where
 
 import Data.UUID.Types (UUID)
 import Hedgehog (MonadGen, forAll, property, (===))
-import Test.Tasty
-import Test.Tasty.Hedgehog
+import Test.Tasty (TestTree)
+import Test.Tasty.Hedgehog (testProperty)
 
-import Lib.Core.Jwt
+import Lib.Core.Jwt (JWTPayload (..), jwtPayloadFromMap, jwtPayloadToMap, jwtUserId)
 
 import qualified Data.UUID.Types as UUID
 import qualified Hedgehog.Gen as Gen
 import qualified Hedgehog.Range as Range
 
 test_jwtMapEncodeAndDecode :: [TestTree]
-test_jwtMapEncodeAndDecode = return $ testProperty "jwtMapEncodeAndDecode" $
-  property $ do
-    randomId <- forAll genRandId
-    let randomJwtPayload = JWTPayload { jwtUserId = randomId }
-    let encoded = jwtPayloadToMap randomJwtPayload
-    let decoded = jwtPayloadFromMap encoded
-    decoded === Just randomJwtPayload
+test_jwtMapEncodeAndDecode = return $ testProperty "jwtMapEncodeAndDecode"
+    $ property $ do
```

Comment:
I mean that you should have `property` function on the same line as `testProperty`. The goal is to reduce amount of lines. Same for PasswordSpec